### ISSUE TITLE
New rule: ARIA required context role

### ISSUE THREAD

**Issue by Jym77** [w3c/wcag-act#499]
The ACT-Rules community group has completed work on "ARIA required context role". We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: .ARIA required context role
**Rule id**: ff89c9
**Link to the rule**: https://act-rules.github.io/rules/ff89c9
**License**: For community groups: W3C Software And Document Notice And License
**Implementors**:
- Alfa (Siteimprove)
- QualWeb (LASIGE, Faculdade de Ciências da Universidade de Lisboa)
- SortSite (PowerMapper)
- axe-core (Deque Systems)

If you'd like to contact me, please e-mail me at jym@siteimprove.com. (optional: You can also reach me on the W3C Community Slack, my name there is Jean-Yves Moyen)

**Comment by maryjom** [comment]
[Survey](https://www.w3.org/2002/09/wbs/93339/ACTARIAContextRole/) open until 17 December.

**Comment by maryjom** [comment]
Discussed during [7 January meeting](https://www.w3.org/2021/01/07-wcag-act-minutes.html#item01).
